ApplicationCode/ProjectDataModel/RimEclipseView.cpp

```diff
@@ -201,6 +207,7 @@ RimEclipseView::RimEclipseView()
 
     m_reservoirGridPartManager = new RivReservoirViewPartMgr( this );
     m_simWellsPartManager      = new RivReservoirSimWellsPartMgr( this );
+    m_streamlinesPartManager   = new RivStreamlinesPartMgr( this );
```

Comment:
See above. It would be good to change all these "new"'s in one go to std::make_unique